tutorials/tmva/TMVAMulticlass.C

```diff
@@ -131,22 +140,36 @@ void TMVAMulticlass( TString myMethodList = "" )
    if (Use["PDEFoam"]) // PDE-Foam approach
       factory->BookMethod( dataloader,  TMVA::Types::kPDEFoam, "PDEFoam", "!H:!V:TailCut=0.001:VolFrac=0.0666:nActiveCells=500:nSampl=2000:nBin=5:Nmin=100:Kernel=None:Compress=T" );
 
-   if (Use["DNN_CPU"]) {
+
+   if (Use["DL_CPU"]) {
       TString layoutString("Layout=TANH|100,TANH|50,TANH|10,LINEAR");
-      TString training0("LearningRate=1e-1, Momentum=0.5, Repetitions=1, ConvergenceSteps=10,"
-                        " BatchSize=256, TestRepetitions=10, Multithreading=True");
-      TString training1("LearningRate=1e-2, Momentum=0.0, Repetitions=1, ConvergenceSteps=10,"
-                        " BatchSize=256, TestRepetitions=7, Multithreading=True");
+      TString training0("LearningRate=1e-3,Momentum=0.5,Repetitions=1,ConvergenceSteps=10,"
+                        "BatchSize=256,TestRepetitions=1");
       TString trainingStrategyString("TrainingStrategy=");
-      trainingStrategyString += training0 + "|" + training1;
+      trainingStrategyString += training0; // + "|" + training1;
       TString nnOptions("!H:V:ErrorStrategy=CROSSENTROPY:VarTransform=N:"
-                        "WeightInitialization=XAVIERUNIFORM:Architecture=CPU");
+                        "WeightInitialization=XAVIERUNIFORM:Architecture=GPU");
       nnOptions.Append(":");
       nnOptions.Append(layoutString);
       nnOptions.Append(":");
       nnOptions.Append(trainingStrategyString);
-      factory->BookMethod(dataloader, TMVA::Types::kDNN, "DNN_CPU", nnOptions);
+      factory->BookMethod(dataloader, TMVA::Types::kDL, "DL_CPU", nnOptions);
    }
+   if (Use["DL_GPU"]) {
+      TString layoutString("Layout=TANH|100,TANH|50,TANH|10,LINEAR");
+      TString training0("LearningRate=1e-3,Momentum=0.5,Repetitions=1,ConvergenceSteps=10,"
+                        "BatchSize=256,TestRepetitions=1");
+      TString trainingStrategyString("TrainingStrategy=");
+      trainingStrategyString += training0;// + "|" + training1;
```

Comment:
Same here, I would remove the `training1` and call `training0` just `training`. Since it's a tutorial we should clean up as much as possible!